sos/report/plugins/openshift_ovn.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2021 Nadia Pinaeva <npinaeva@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class OpenshiftOVN(Plugin, RedHatPlugin):
+    """This plugin is used to collect OCP 4.x OVN logs.
+    """
+    short_desc = 'Openshift OVN'
+    plugin_name = "openshift_ovn"
+    containers = ('ovnkube-master', 'ovn-ipsec')
+    profiles = ('openshift',)
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/lib/ovn/etc/ovnnb_db.db",
+            "/var/lib/ovn/etc/ovnsb_db.db",
```

Comment:
These is already collected by `ovn_central` plugin (https://github.com/sosreport/sos/blob/main/sos/report/plugins/ovn_central.py#L135-L141). We should collect any file by no more than 1 plugin.

Reply:
Yeah, so the difference is that this plugin is used to collect ovn logs from container, and these pathes are actually just where container mounts its directories.
I'd prefer to have `add_copy_spec_container` to copy files directly from containers, so that changing host mount directory doesn't break everything, but with `crictl` it's not possible to copy files from containers https://github.com/kubernetes-sigs/cri-tools/issues/677, and using `kubectl cp` also have limitations.

About collecting files only once, we have this problem about using the same piece of software but in different "configurations" for different systems. Files I'm trying to collect are openshift-specific (and plugin is named accordingly) meaning that code that configures ovn and defines where required files will be located is a part of OpenShift code.
It also means that these plugins for ovn are mutually exclusive, so these same files will never be collected twice (if system configuration makes any sense ofc).

There is also an example of collecting the same files in different plugins: `ovn_host` and `ovn_central` copying the same logs: https://github.com/sosreport/sos/blob/main/sos/report/plugins/ovn_host.py#L33-L36 and https://github.com/sosreport/sos/blob/main/sos/report/plugins/ovn_central.py#L82-L85, but I think these plugins are also never supposed to be active on the same host

Reply:
OK